src/project-tests/quote-machine-tests.js

```diff
@@ -96,6 +96,8 @@ export default function createRandomQuoteMachineTests() {
               }, 500);
             });
           });
+        } else {
+          assert.fail('element with id="text" not found');
```

Comment:
Personally, I would have gone with asserting on the element we are testing:
```js
assert.exists(text);
```
Same goes for other tests. Thoughts?

Reply:
Shouldn't it be `notExists` then as we are expecting it to be falsy in the else block?

Anyway, I guess it's more explicit so I'm fine with changing it.

Reply:
To clarify, there should be no `else` clause here - we should assert that the element exists. The test **should** fail, if it does not exist.

Reply:
We already have tests for the elements, tests 2 and 3. The two tests (6 and 7) are testing for the text content of the element but never fails if the elements are null.

Reply:
Also, a user brought up a good point - we should add this test to all that do not have it: https://forum.freecodecamp.org/t/build-a-random-quote-machine-bug-in-test/472095/3